### ISSUE TITLE
Adding AI vs AI demo game mode as in the original Version

### ISSUE THREAD

**Issue by Wakeboardtiger** [nicymike/serflings#88]
Just noticed that this option is missing as i wanted to use it, for research purposes with the 64x speed mode :)

**Comment by nicymike** [comment]
Yes, there is some stuff missing for demo mode eg. allow to check every building but don't allow to build.

**Comment by nicymike** [comment]
Added the feature as preview. But be warned, some things are not working as expected yet.

**Comment by Wakeboardtiger** [comment]
Sorry, how can i activate it, I dont seem to be able to find it.
ahh found the sompelton solution: https://github.com/nicymike/serflings/issues/143

I see i can increase the mapsize to 11? Is this real or a mistake, is it realy makeing it that big?

**Comment by nicymike** [comment]
Map size 11 is also a preview feature and will be available in preview mode. It is indeed increasing the map size to 11.
Same warning as for demo mode: The game may break with map size 11.

- 1 -   32 x   32
- 2 -   64 x   32
- 3 -   64 x   64
- 4 -  128 x   64
- 5 -  128 x  128
- 6 -  256 x  128
- 7 -  256 x  256
- 8 -  512 x  256
- 9 -  512 x  512 (preview mode)
- 10 - 1024 x  512 (preview mode)
- 11 - 1024 x 1024 (preview mode)

**Comment by Wakeboardtiger** [comment]
oh cool, thanks
